var/spack/repos/builtin/packages/py-stdlib-list/package.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+class PyStdlibList(PythonPackage):
+    """This package includes lists of all of the standard libraries
+    for Python, along with the code for scraping the official Python
+    docs to get said lists."""
+
+    homepage = "https://pypi.python.org/project/stdlib-list"
+    url      = "https://pypi.io/packages/source/s/stdlib-list/stdlib-list-0.5.0.tar.gz"
+
+    version('0.6.0', sha256='133cc99104f5a4e1604dc88ebb393529bd4c2b99ae7e10d46c0b596f3c67c3f0')
+    version('0.5.0', sha256='5311a95812ebce3c5ad0b1f6ded798ce6f945d157075d166a5426c2da75a6625')
+
+    depends_on('py-functools32', when="^python@:3.1", type=('build', 'run'))
```

Comment:
Where do you see this dependency listed?

Reply:
setup.py, "install_requires",
also:
```
coltrane (/tmp/stdlib-list-0.6.0):pydeps --external ./stdlib_list
[
    "functools32", 
    "sphinx"
]
['functools32', 'sphinx']

```

Reply:
Ah, in that case:
```suggestion
    depends_on('py-functools32', when="@0.6.0: ^python@:3.1", type=('build', 'run'))
```
or just remove version 0.5.0.